systems/data-plan/base-rate/pkg/validations/validations.go

```diff
@@ -17,10 +18,23 @@ func IsValidUploadReqArgs(fileUrl, effectiveAt, simType string) bool {
 	return !IsEmpty(fileUrl, effectiveAt, simType)
 }
 
-func IsFutureDate(date string) bool {
-	t, err := time.Parse(time.RFC3339, date)
-	if err != nil {
-		return false
-	}
-	return time.Now().Before(t)
+
+
+func IsFutureDate(date string) error {
+    t, err := time.Parse(time.RFC1123, date)
```

Comment:
Idea is to have to have time methods in common package and use it from there. But I think @delvatt  is working on that. May be once he is done will update this too.